src/client/client.js

```diff
@@ -49,6 +45,9 @@ const createCashay = (store, cashaySchema) => {
     // ENABLE THIS FOR EXPLORING FRONT END PERFORMANCE
     // const {whyDidYouUpdate} = require('why-did-you-update');
     // whyDidYouUpdate(React);
+    // ENABLE THIS AS ANOTHER MEANS OF TESTING FRONT END PERFORMANCE
+    // const Perf = require('react-addons-perf');
+    // window.Perf = Perf;
```

Comment:
Everybody knows that only I can optimize this application. When I start to optimize this application, it's going to move so fast it's going to make the user's head spin. Don't listen to what other people might tell you. Those people are stupid. Hey, is that a baby? Would somebody through that baby outta here?